ossdbtoolsservice/language/language_service.py

```diff
@@ -69,7 +69,8 @@ def __init__(self):
         self._logger: [Logger, None] = None
         self._provider_valid_uri: Dict[str, Set] = {
             utils.constants.PG_PROVIDER_NAME: set(), 
-            utils.constants.MYSQL_PROVIDER_NAME: set()
+            utils.constants.MYSQL_PROVIDER_NAME: set(),
+            utils.constants.MSSQL_PROVIDER_NAME: set()
         }
```

Comment:
This should actually be others as there can be some other flavors that we do not know about, we only care for the ones that we support. So all others should go in that dictionary.

Reply:
Do you mean that I should replace the mssql key in the dictionary with "others"?